buildroot/share/PlatformIO/scripts/common-dependencies.py

```diff
@@ -15,7 +15,7 @@
 	FEATURE_CONFIG = {}
 
 	def validate_pio():
-		PIO_VERSION_MIN = (5, 0, 3)
+		PIO_VERSION_MIN = (6, 0, 1)
 		try:
 			from platformio import VERSION as PIO_VERSION
```

Comment:
FYI, the PIO_VERSION can contain a development build in the PATCH part. For example, 

```
PIO_VERSION = (6, 0, "2a1")
```

Will it break your verification?

Reply:
Users should not be building firmware with alpha tools.

Reply:
> Will it break your verification?

It might, if we are trying to do an older/newer version comparison.

Reply:
The `re.sub` ensures only the numeric part is used, so our < comparison is good.